test/integration/consul-container/libs/cluster/container.go

```diff
@@ -561,6 +602,7 @@ func newContainerRequest(config Config, opts containerOpts, ports ...int) (podRe
 		ExposedPorts: []string{
 			"8500/tcp", // Consul HTTP API
 			"8501/tcp", // Consul HTTPs API
+			"8502/tcp", // Consul gRPC API
```

Comment:
Regarding grpc TLS, wouldn't port 8503 need to be exposed here?

Reply:
Yep! Didn't include right now, as it wasn't clear how to get the test harness to use gRPC TLS anway.